modules/apps/release-feature-flag/release-feature-flag-api/src/main/java/com/liferay/release/feature/flag/ReleaseFeatureFlag.java

```diff
@@ -19,6 +19,6 @@
  */
 public enum ReleaseFeatureFlag {
 
-	DISABLE_PRIVATE_LAYOUTS
+	DISABLE_PRIVATE_LAYOUTS, ENABLE_GOOGLE_DRIVE_SHORTCUTS
```

Comment:
Any suggestion? Having one of them being "Enable" and the other "Disable" is weird, but technically, ours is the good one: feature flags are meant to be used to enable new features.

Reply:
One thing.

By default the GDShortcut will be disabled or enabled.

Maybe it's the page of the Release Feature Flags the one to be revisited.
maybe to add more info because it's confusing